independent-publisher-connectors/Etsy/apiDefinition.swagger.json

```diff
@@ -443,9 +742,40 @@
     "/shops/{shop_id}/receipts/{receipt_id}/transactions": {
       "get": {
         "responses": {
-          "default": {
-            "description": "default",
-            "schema": {}
+          "200": {
+            "description": "A list of transactions.",
+            "schema": {
+              "$ref": "#/definitions/ShopReceiptTransactions"
+            },
+            "headers": {}

```

Comment:
@troystaylor Since headers has empty definition, it can be removed. Do you have any specific reason to provide empty headers definition ?

Reply:
These were undefined artifacts from the source OpenAPI file.